src/odemis/driver/test/hamamatsurx_test.py

```
@@ -718,38 +756,133 @@ def test_acqSync_EarlyEvents(self):
-            self.camera_left -= 1
-            logging.debug("Got image.")
-            if self.camera_left <= 0:
-                dataflow.unsubscribe(receive_image)
+        self.images_left = num_images
+        self._expected_shape = size[::-1]  # Y, X
 
-        self.readoutcam.data.subscribe(receive_image)
+        self.readoutcam.data.subscribe(self.receive_image)
 
         # Wait for the image
         for i in range(num_images):
```

Ruff (0.16.1)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)